eurocodedesign/units/__init__.py

```diff
@@ -41,6 +43,7 @@ class PhysicalType(Enum):
     ACCELERATION = auto()
     MASS = auto()
     FORCE = auto()
+    TENSION = auto()
```

Comment:
Tension is not the right term for this. I would maybe call it a `LINEFORCE `or `LINEALFORCE`

Reply:
I rename it to STIFFNESS